mis_template_financial_report/models/mis_report_instance.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Hunki Enterprises BV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import copy
+from collections import OrderedDict
+
+from odoo import api, fields, models
+
+
+class MisReportInstance(models.Model):
+    _inherit = "mis.report.instance"
+
+    allow_horizontal = fields.Boolean(compute="_compute_allow_horizontal")
+    horizontal = fields.Boolean()
+
+    @api.depends("report_id")
+    def _compute_allow_horizontal(self):
```

Comment:
Shouldn't this be in an isolated module or part of the main one?

Reply:
As apparent from the issues that @hbrunn linked to, that is a complex topic. I've mentioned it here now https://github.com/OCA/mis-builder/issues/389 in the context of the 15.0 migration of mis_builder. Out of scope for this version I believe.